### ISSUE TITLE
SVGTextPathElement interface seems to be missing properties

### ISSUE THREAD

**Issue by longsonr** [w3c/svgwg#391]
Please:
* https://www.w3.org/TR/SVG2/text.html#InterfaceSVGTextPathElement

Shouldn't it have path and side properties?

**Comment by AmeliaBR** [comment]
I agree that, for consistency, the new attributes should be reflected in properties.

That said, it's not straightforward how to do so.

For `side`, do we create new named constants to expose it as an `SVGAnimatedEnumeration`, or expose it as a modern WebIDL enum?  (Since there are already two other enumerations on the class, I'd sadly have to vote for using something `const unsigned short TEXTPATH_SIDE_LEFT= 1`)

For `path`, there is no standard way of exposing path data as DOM properties in SVG 2. The `d` property is not exposed on `SVGPathElement`. The SVG 1 interfaces to path data were removed, and the replacements are languishing in the [Paths module](https://svgwg.org/specs/paths/#DOMInterfaces).

(But, that said, you'd have to dig through the old www-svg discussions to confirm whether there was an intentional decision not to expose these attributes as DOM properties, or if it was just an oversight.)

**Comment by boggydigital** [comment]
Not blocking updated 2.0 CR publication - assigning 2.1 WD milestone
